howso/ablation.amlg

```diff
@@ -222,10 +243,7 @@
 
 		(assign (assoc
 			max_influence_weight_entropy_to_keep
-				(compute_on_contained_entities (list
-					(query_exists !internalLabelInfluenceWeightEntropy)
-					(query_quantile !internalLabelInfluenceWeightEntropy !autoAblationInfluenceWeightEntropyThreshold ".case_weight")
-				))
+				(call !MaxInfluenceWeightEntropyToKeepQuery)
```

Comment:
Not specifying the weight feature name here feels a bit scary. Is the autoAblationWeightFeature name changeable by the user (via set_auto_ablation_params, I would assume if so)

Reply:
Yeah, it is.